Wikipedia/Code/WMFImageTag.m

```diff
@@ -84,9 +88,14 @@ - (instancetype)initWithImageTagContents:(NSString*)imageTagContents {
         *stop = width && height && dataFileWidth && dataFileHeight && src && srcAttributeRange.location != NSNotFound;
     }];
 
+    //Don't continue initialization if we have invalid src
+    if([src length] == 0){
```

Comment:
Should this be doing the same `if([[src stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]] length] == 0){` check?